### DISCUSSION TITLE
Kotlin actual/expect file name pattern support

### DISCUSSION THREAD

**Discussion by Savrov** [detekt/detekt#7068] [Q&A] [answered]
Hello, I'm missing for support of Kotlin actual/expect file name pattern support. Noticed that it was implemented there: https://github.com/detekt/detekt/pull/6426

https://github.com/detekt/detekt/blob/7281dd5a23f73b528c91df9970ee0b2275bb01db/detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/MatchingDeclarationName.kt#L48

But it did not get to the main branch yet:
https://github.com/lexa-diky/detekt/blob/main/detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/MatchingDeclarationName.kt

So, the questing is, when we can expect it to be released?

**Comment by schalkms** [comment, marked as answer]
Linked PR was merged to main. However, it's a breaking change and thus will only be included in detekt v2.0 and upwards, but not in v1.x.
